XMonad/Doc/Extending.hs

```diff
@@ -154,23 +147,23 @@ In the @XMonad.Hooks@ namespace you can find modules exporting
 hooks. Hooks are actions that xmonad performs when certain events
 occur. The three most important hooks are:
 
-* 'XMonad.Core.manageHook': this hook is called when a new window
+* 'XMonad.Core.manageHook': this hook is called when a new window that
   xmonad must take care of is created. This is a very powerful hook,
   since it lets us examine the new window's properties and act
   accordingly. For instance, we can configure xmonad to put windows
   belonging to a given application in the float layer, not to manage
   dock applications, or open them in a given workspace. See
-  "XMonad.Doc.Extending#Editing_the_manage_hook" for more information on
-  customizing 'XMonad.Core.manageHook'.
+  "XMonad.Doc.Extending#Editing_the_manage_hook" for more information
+  on customizing 'XMonad.Core.manageHook'.
 
 * 'XMonad.Core.logHook': this hook is called when the stack of windows
-  managed by xmonad has been changed, by calling the
-  'XMonad.Operations.windows' function. For instance
+  managed by xmonad has been changed; for example, this is invoked at
+  the end of the 'XMonad.Operations.windows' function. For instance
   "XMonad.Hooks.DynamicLog" will produce a string (whose format can be
```

Comment:
Do we want to use XMonad.Hooks.StatusBar here?

Reply:
We probably want to rewrite the whole `logHook` section to instead refer to X.H.StatusBar, but just switching the names won't do (because the functions referred to in these sections are specifically functions from X.H.DynamicLog)

Reply:
I created https://github.com/xmonad/xmonad-contrib/issues/645 to keep track of this